applications/HDF5Application/python_scripts/temporal_output_process_factory.py

```diff
@@ -14,6 +15,7 @@ def Execute(self, settings, Model):
         settings = settings.Clone()
         settings.ValidateAndAssignDefaults(default_settings)
         model_part = Model[settings["model_part_name"].GetString()]
+        file_utilities.CheckIfWriteInFolder(settings, model_part)
```

Comment:
this is used in case one wants to write the h5 files to a folder
=> check the `file_utilities` to see how it works